### PR TITLE
Chore.upgrade applicationinsights v3

### PR DESCRIPTION
**Pull request by jannisvisser**
[AB#31413](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/31413)

## Describe your changes

Attempt nr 2 at upgrading applicationinsights package.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added tests wherever relevant
- [ ] I have made sure that all automated checks pass before requesting a review
- [ ] I do not need any deviation from our PR guidelines

### CODE REVIEW

**Comment by jannisvisser** [comment]
@PeterSmallenbroek see last commit, which is the only new one compared to previous PR. This fixes it for me locally when using the 121-staging connection string. And does lead to logs on 121-staging when using the mock-nlrc-reset endpoint locally.

I'm in other meetings now, so if you can already review this. Not sure if we can do this in a nicer way without the require import..

File: services/121-service/src/shared/middleware/azure-logger.middleware.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import { Injectable, NestMiddleware } from '@nestjs/common';
 import { TelemetryClient } from 'applicationinsights';
```
Comment (PeterSmallenbroek) [inline services/121-service/src/shared/middleware/azure-logger.middleware.ts:2]:
```suggestion
import { defaultClient, TelemetryClient } from 'applicationinsights';
```

File: services/121-service/src/shared/middleware/azure-logger.middleware.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 import { NextFunction, Request, Response } from 'express';
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+import appInsights = require('applicationinsights');
```
Comment (PeterSmallenbroek) [inline services/121-service/src/shared/middleware/azure-logger.middleware.ts:5]:
```suggestion
```

File: services/121-service/src/shared/middleware/azure-logger.middleware.ts [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 export class AzureLoggerMiddleware implements NestMiddleware {
   defaultClient: TelemetryClient;
 
   constructor() {
-    if (process.env.APPLICATION_INSIGHT_IKEY) {
-      this.defaultClient = new TelemetryClient(
-        process.env.APPLICATION_INSIGHT_IKEY,
-      );
+    if (process.env.APPLICATIONINSIGHTS_CONNECTION_STRING) {
+      this.defaultClient = appInsights.defaultClient;
```
Comment (PeterSmallenbroek) [inline services/121-service/src/shared/middleware/azure-logger.middleware.ts:13]:
```suggestion
      this.defaultClient = defaultClient;
```

**Review by PeterSmallenbroek — CHANGES_REQUESTED** [review]
These changes might be what you want to get rid of the require. Can you confirm/test? (Should also still be replicated in the other place)

**Review by PeterSmallenbroek — APPROVED** [review]

**Pull request merged**